feature_tests/features/consumer/consumer-readDocumentPointer-edge.feature

```diff
@@ -87,9 +87,9 @@ Feature: Consumer Read Edge Case scenarios
     When Consumer "Yorkshire Ambulance Service" reads an existing Document Reference "8FW23-1114567891"
     Then the operation is unsuccessful
```

Comment:
Can we please add the step to test the status code to all the tests in the PR where it is missing - **And the status is '400'.**

Reply:
Yeah can do :)